api/services/SurveyResultService.js

```diff
@@ -146,14 +147,14 @@ module.exports = class SurveyResultService extends Service {
                 return a;
             }, []);
 
-            return matches.map(m => {
+            return orderBy(matches.map(m => {
```

Comment:
What is the reason behind moving the sort to the service?

Reply:
I needed to move sorting out of the results component to get the right info for social sharing, but the sorting could also happen in result-actions or result-reducer

Reply:
That would be better. It is cleaner design not to assume any ordering of the data returned by the API.